### DISCUSSION TITLE
BagPart - ContentCard - create a template for a ContentType

### DISCUSSION THREAD

**Discussion by PiemP** [OrchardCMS/OrchardCore#9558] [Q&A] [answered]
Hi all, 

I'm trying to create a template to render the ContentCard title without the ContentType display name like this:
![boxlink_linkwithicon](https://user-images.githubusercontent.com/2736009/119798934-e55b1900-bedb-11eb-975f-da282d839526.jpg)

I have already an admin theme customized and I have created almost all the template name I can imagine but no one is called:
```
BagPart-ContentCard-BagPart.Edit.cshtml
BoxLink-BagPart-ContentCard-BagPart-Edit.Display.cshtml
BoxLink-BagPart-ContentCard-BagPart.Display.cshtml
BoxLink-BagPart-ContentCard-BagPart.Edit.cshtml
BoxLink-BagPart-ContentCard-Edit.Display.cshtml
BoxLink-BagPart-ContentCard-Edit.Display.Edit.cshtml
BoxLink-BagPart-ContentCard.Display.Edit.cshtml
BoxLink-BagPart-ContentCard.Edit.cshtml
BoxLink-BagPart-Edit.Display.cshtml
BoxLink-ContentCard-BagPart-BagPart.Edit.cshtml
BoxLink-ContentCard-BagPart.Edit.cshtml
```
How I can customize the ContentCard template for a specific ContentType?

The ContentType name is **Boxlink**.
The BagPart name is **BagPart**.

Thank you.

**Comment by jtkech** [comment, marked as answer]
Regarding the code

    //Card Shape
    private const string ContentCardEdit = "ContentCard_Edit";

    //Define edit card shape per content type per collection type
    //ContentCard_Edit__[CollectionType]__[ContentType]
    //e.g. ContentCard_Edit__FlowPart__Paragraph, ContentCard_Edit__BagPart__Form, ContentCard_Edit__FlowPart__Input
    contentCardEditor.Metadata.Alternates.Add($"{ContentCardEdit}__{collectionType}__{contentType}");


Try `ContentCard-BagPart-BoxLink.Edit.cshtml`

**Reply by PiemP** [reply]
Thank you @jtkech . Now I have a tip to better locate code to understand this functionality 😉.
